crates/bevy_window/src/event.rs

```diff
@@ -133,6 +133,9 @@ pub struct CursorMoved {
     pub window: Entity,
     /// The cursor position in logical pixels.
     pub position: Vec2,
+    /// The change in the position of the cursoe since the last event was sent.
+    /// This value is `None` if the cursor was outside the window area during the last frame.
+    pub delta: Option<Vec2>,
```

Comment:
Could you add a note/warning inspired by [this one](https://docs.rs/winit/latest/winit/event/enum.WindowEvent.html#variant.CursorMoved.field.position) from the winit doc explaining not to rely on this for non "cursor-like" mouse-based input and to use `MouseMotion` instead?

Reply:
```suggestion
    //
    // Because the range of this data is limited by the display area and it may have been 
    //  transformed by the OS to implement effects such as cursor acceleration, it should 
    // not be used to implement non-cursor-like interactions such as 3D camera control.
    pub delta: Option<Vec2>,
```